src/service/deletion-repeater.ts

```diff
@@ -43,11 +43,25 @@ export interface DeletionObservable {
 export class DeletionRepeater<M extends DeletionObservable>
   implements MessageEventResponder<M>
 {
+  /**
+   * メッセージを無視するかどうかを判定する述語。
+   * この述語がtrueを返した場合、内容を復唱しない。
+   */
+  private isIgnoreTarget: (content: string) => boolean;
+
+  constructor(isIgnoreTarget: (content: string) => boolean) {
+    this.isIgnoreTarget = isIgnoreTarget;
+  }
```

Comment:
```suggestion
  constructor(private readonly isIgnoreTarget: (content: string) => boolean) {}
```
TypeScript の機能でまとめておくとよいでしょう.